stdlib/private/DifferentiationUnittest/DifferentiationUnittest.swift

```diff
@@ -257,18 +257,27 @@ extension Tracked where T : Differentiable & FloatingPoint, T == T.TangentVector
   }
 }
 
-// Differential operators for `Tracked<Float>`.
-public func gradient(
-  at x: Tracked<Float>, in f: @differentiable (Tracked<Float>) -> Tracked<Float>
-) -> Tracked<Float> {
-  return pullback(at: x, in: f)(1)
+// Differential operators for `Tracked<T>`.
+
+public func gradient<T, U>(
```

Comment:
IIRC,`Tracked: FloatingPoint` conformance caused `*` operator lookup for `@differentiating(*)` to become ambiguous.

That seems workaround-able by using `@differentiable(vjp: ...)` for now. We should probably investigate fixing `@differentiating(*)` ambiguous lookup (and `@differentiating` original declaration lookup for initializers/subscripts/properties) sometime.

Reply:
@rxwei, yes it would be good to make `Tracked` conform to `FloatingPoint`, but have issues that Dan mentions. I had already filed a bug: https://bugs.swift.org/browse/TF-926